en/gps_compass/rtk_gps.md

```diff
@@ -144,9 +146,16 @@ The RTK GPS connection is essentially plug and play:
 
 ### Configuring GPS as Yaw/Heading Source
 
-GPS can be used as a source for yaw fusion when using modules where _yaw output is supported by the device_ (e.g. [Trimble MB-Two](../gps_compass/rtk_gps_trimble_mb_two.md)) or when using some [RTK GPS Setups with Dual u-blox F9P](../gps_compass/u-blox_f9p_heading.md).
+GPS can be used as a source for yaw fusion when using a single device with two antenna where _yaw output is supported by the device_, or when using some [RTK GPS Setups with Dual u-blox F9P](../gps_compass/u-blox_f9p_heading.md).
+Using GPS as a heading source has the benefit that yaw calculations are not impacted by magnetic interference.
+
+Both approaches work comparing the time taken for a GNSS signal to reach two separated antennas.
+The minimum distance between antenna depends on the device but is of the order of 50 cm (check manufacturer documentation).
+
+The devices that can be used are listed in this way are listed in the **GPS Yaw** column of the table above, such as [Septentrio AsteRx-m3 Pro](../gps_compass/septentrio_asterx-rib.md), [Holybro H-RTK Unicore UM982 GPS](../gps_compass/rtk_gps_holybro_unicore_um982.md), and [Trimble MB-Two](../gps_compass/rtk_gps_trimble_mb_two.md).
+The links in the table take you to the device-specific PX4 configuration.
 
-When using GPS for yaw fusion you will need to configure the following parameters:
+Generally when using a GNSS as a source of yaw information you will need to configure the following parameters:
```

Comment:
So the params below seem to be used by Unicore and by dual FP9 setups. Septentrio have a different setup. It isn't clear if these _also_ need to be set for septentrio. Because of that I point to the device specific docs too.

Hope this makes sense @julianoes